File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--creating-shortcut-menus-for-bookmarks_3.cs
        public bool GetVisible(Office.IRibbonControl control)
        {
            if (control.Id == "BoldButton")
            {
                if (Globals.ThisDocument.showBoldButton == true)
                {
                    Globals.ThisDocument.showBoldButton = false;
                    return true;
                }
                else
                {
                    return false;
                }
            }
            else if (control.Id == "ItalicButton")
            {
                if (Globals.ThisDocument.showItalicButton == true)
                {
                    Globals.ThisDocument.showItalicButton = false;
                    return true;
                }
                else
                {
                    return false;
                }
            }
            else
            {
                return false;
            }
       
        }
 

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--creating-shortcut-menus-for-bookmarks_3.vb
    Public Function GetVisible(ByVal control As Office.IRibbonControl) As Boolean
        If control.Id = "BoldButton" Then
            If Globals.ThisDocument.ShowBoldButton = True Then
                Globals.ThisDocument.ShowBoldButton = False
                Return True
            Else
                Return False
            End If
        ElseIf control.Id = "ItalicButton" Then
            If Globals.ThisDocument.ShowItalicButton = True Then
                Globals.ThisDocument.ShowItalicButton = False
                Return True
            Else
                Return False
            End If
        Else
            Return False
        End If
    End Function